[2017-08-17 22:33:47,591] ERROR Securitisation.Utilities.Common.Log4NetLogService - Run task for 47e455bc-94ba-4899-8945-62836247782b failed
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.ArgumentNullException: Value cannot be null.
Parameter name: Can not find [DefaultCurve] from Session Context Array.
The given key was not present in the dictionary.
   at DADP.CashFlowStudioEntity.CashFlowInputProvider.InitSessionContextArrayFromMemory(String sSessionId, String sConnectionString, String sAppDomain, Dictionary`2& sessionContext, Dictionary`2& sessionContextArray)
   --- End of inner exception stack trace ---
   at BL.TaskProcessService.CashFlowProvider.RunManagedMethodByPath(ITaskAction action)
   --- End of inner exception stack trace ---
   at BL.TaskEngine.TaskEngine.RunProcessesByTask()
   at TaskProcessEngine.CashflowService.RunTask(Guid sessionId, String applicationDomain) in e:\Dev\SFM\Products\TaskProcessEngine\TaskProcessEngine\CashflowService.svc.cs:line 64
[2017-08-17 22:41:46,942] ERROR Securitisation.Utilities.Common.Log4NetLogService - Run task for c4242ff8-11cd-4107-ad2a-f6fd117d7656 failed
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.ThrowHelper.ThrowKeyNotFoundException()
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at DADP.CashFlowStudioEntity.CashFlowInputProvider.ReadTaskArrayFromDataBase(String sCashflowName, String sSessionId, String sConnectionString, String sAppDomain, Dictionary`2& sessionContext, Dictionary`2& sessionContextArray)
   --- End of inner exception stack trace ---
   at BL.TaskProcessService.CashFlowProvider.RunManagedMethodByPath(ITaskAction action)
   --- End of inner exception stack trace ---
   at BL.TaskEngine.TaskEngine.RunProcessesByTask()
   at TaskProcessEngine.CashflowService.RunTask(Guid sessionId, String applicationDomain) in e:\Dev\SFM\Products\TaskProcessEngine\TaskProcessEngine\CashflowService.svc.cs:line 64
[2017-08-17 22:43:02,252] ERROR Securitisation.Utilities.Common.Log4NetLogService - Run task for d979c157-d387-4ec5-84b9-1816b5fa605f failed
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.ArgumentNullException: Value cannot be null.
Parameter name: Can not find [DefaultCurve] from Session Context Array.
The given key was not present in the dictionary.
   at DADP.CashFlowStudioEntity.CashFlowInputProvider.InitSessionContextArrayFromMemory(String sSessionId, String sConnectionString, String sAppDomain, Dictionary`2& sessionContext, Dictionary`2& sessionContextArray)
   --- End of inner exception stack trace ---
   at BL.TaskProcessService.CashFlowProvider.RunManagedMethodByPath(ITaskAction action)
   --- End of inner exception stack trace ---
   at BL.TaskEngine.TaskEngine.RunProcessesByTask()
   at TaskProcessEngine.CashflowService.RunTask(Guid sessionId, String applicationDomain) in e:\Dev\SFM\Products\TaskProcessEngine\TaskProcessEngine\CashflowService.svc.cs:line 64
[2017-08-17 22:45:17,255] ERROR Securitisation.Utilities.Common.Log4NetLogService - Run task for 1d77c4a7-41f6-4d5f-9909-54564f877059 failed
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.ArgumentNullException: Value cannot be null.
Parameter name: Can not find [DefaultCurve] from Session Context Array.
The given key was not present in the dictionary.
   at DADP.CashFlowStudioEntity.CashFlowInputProvider.InitSessionContextArrayFromMemory(String sSessionId, String sConnectionString, String sAppDomain, Dictionary`2& sessionContext, Dictionary`2& sessionContextArray)
   --- End of inner exception stack trace ---
   at BL.TaskProcessService.CashFlowProvider.RunManagedMethodByPath(ITaskAction action)
   --- End of inner exception stack trace ---
   at BL.TaskEngine.TaskEngine.RunProcessesByTask()
   at TaskProcessEngine.CashflowService.RunTask(Guid sessionId, String applicationDomain) in e:\Dev\SFM\Products\TaskProcessEngine\TaskProcessEngine\CashflowService.svc.cs:line 64
[2017-08-17 22:50:30,263] ERROR Securitisation.Utilities.Common.Log4NetLogService - Run task for 21300dfa-3172-4fc7-ab36-9fdffe754130 failed
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.ArgumentNullException: Value cannot be null.
Parameter name: Can not find [DefaultCurve] from Session Context Array.
The given key was not present in the dictionary.
   at DADP.CashFlowStudioEntity.CashFlowInputProvider.InitSessionContextArrayFromMemory(String sSessionId, String sConnectionString, String sAppDomain, Dictionary`2& sessionContext, Dictionary`2& sessionContextArray)
   --- End of inner exception stack trace ---
   at BL.TaskProcessService.CashFlowProvider.RunManagedMethodByPath(ITaskAction action)
   --- End of inner exception stack trace ---
   at BL.TaskEngine.TaskEngine.RunProcessesByTask()
   at TaskProcessEngine.CashflowService.RunTask(Guid sessionId, String applicationDomain) in e:\Dev\SFM\Products\TaskProcessEngine\TaskProcessEngine\CashflowService.svc.cs:line 64
